### PR TITLE
CORCI-1081 ci: Move Unit Testing from EL7 to EL8

### PR DESCRIPTION
**Pull request by brianjmurrell**

### CODE REVIEW

File: ci/provisioning/post_provision_config_nodes_EL_8.sh
Comment (brianjmurrell) [reply ci/provisioning/post_provision_config_nodes_EL_8.sh:13]: It's a single long string.

File: Jenkinsfile
Comment (phender) [inline Jenkinsfile:809]: Is this a typo?
```suggestion
                } // stage('Test CentOS 8.3.2011 RPMs')
```

File: Jenkinsfile
Comment (phender) [inline Jenkinsfile:920]: Should these match?
```suggestion
                } // stage('Fault inection testing on CentOS 8')
```

File: Jenkinsfile
Comment (phender) [inline Jenkinsfile:626]: Should these match?
```suggestion
                } // stage('Unit test Bullseye on CentOS 8')
```

File: Jenkinsfile
Comment (phender) [inline Jenkinsfile:648]: Should these match?
```suggestion
                } // stage('Unit Test with memcheck on CentOS 8')
```

File: Jenkinsfile
Comment (phender) [inline Jenkinsfile:1006]: I cannot select line 1033 to add the comment there, but should these comments match, e.g.
```
} // stage('Bullseye Report on CentOS 8')
```

**Review by phender — CHANGES_REQUESTED** [review]

**Review by jolivier23 — DISMISSED** [review]

File: Jenkinsfile
Comment (brianjmurrell) [reply Jenkinsfile:809]: Yeah.  It's in a comment, so benign.

File: Jenkinsfile
Comment (brianjmurrell) [reply Jenkinsfile:920]: Probably, but also a comment that can be cleaned up in a subsequent PR (and might be in a PR I have in progress even).

File: Jenkinsfile
Comment (brianjmurrell) [reply Jenkinsfile:626]: Ditto.

File: Jenkinsfile
Comment (brianjmurrell) [reply Jenkinsfile:1006]: Ideally.

Is it worth holding up a 2.0 blocker to fix these comments?  I will respin if you feel like they need fixing in this PR.

Comment (brianjmurrell) [reply Jenkinsfile:1006]: @phender This PR is not as much of a long-pole as I understood it to be.  I will fix the comments and re-push.

File: ci/provisioning/post_provision_config_nodes_EL_8.sh
Comment (brianjmurrell) [reply ci/provisioning/post_provision_config_nodes_EL_8.sh:13]: Long string.

File: Jenkinsfile
Comment (phender) [reply Jenkinsfile]: Ah, true.  Then these could have been comments with approval.

**Review by phender — APPROVED** [review]

File: src/include/daos_fs.h
Comment (jolivier23) [inline src/include/daos_fs.h:2]: I just did a copyright update patch so if you merge latest master, that should cut down on these.

**Review by jolivier23 — CHANGES_REQUESTED** [review]

**Comment by mchaarawi** [comment]
please close or convert to draft till it's ready

**Pull request closed without merging**
